next/TilesetWithFullMetadata/TilesetWithFullMetadata.json

```diff
@@ -0,0 +1,1318 @@
+          "example_fixed_length_ARRAY_normalized_UINT16" : [ 0, 16383, 32767, 49151, 65535 ],
+          "example_variable_length_ARRAY_INT32" : [ -2147483648, -715827883, 715827882, 2147483647 ],
+          "example_variable_length_ARRAY_normalized_INT32" : [ -2147483648, -715827883, 715827882, 2147483647 ],
+          "example_fixed_length_ARRAY_INT32" : [ -2147483648, -1073741824, 0, 1073741823, 2147483647 ],
+          "example_fixed_length_ARRAY_normalized_INT32" : [ -2147483648, -1073741824, 0, 1073741823, 2147483647 ],
+          "example_variable_length_ARRAY_UINT32" : [ 0, 1431655764, 2863311530, 4294967295 ],
+          "example_variable_length_ARRAY_normalized_UINT32" : [ 0, 1431655764, 2863311530, 4294967295 ],
+          "example_fixed_length_ARRAY_UINT32" : [ 0, 1073741823, 2147483647, 3221225471, 4294967295 ],
+          "example_fixed_length_ARRAY_normalized_UINT32" : [ 0, 1073741823, 2147483647, 3221225471, 4294967295 ],
+          "example_variable_length_ARRAY_INT64" : [ -9223372036854775808, -3074457346233150072, 3074457346233150071, 9223372036854775807 ],
+          "example_variable_length_ARRAY_normalized_INT64" : [ -9223372036854775808, -3074457346233150072, 3074457346233150071, 9223372036854775807 ],
+          "example_fixed_length_ARRAY_INT64" : [ -9223372036854775808, -4611686018427387904, 0, 4611686018427387903, 9223372036854775807 ],
+          "example_fixed_length_ARRAY_normalized_INT64" : [ -9223372036854775808, -4611686018427387904, 0, 4611686018427387903, 9223372036854775807 ],
+          "example_variable_length_ARRAY_UINT64" : [ 0, 6148914690621625735, 12297829383087925879, 18446744073709551615 ],
+          "example_variable_length_ARRAY_normalized_UINT64" : [ 0, 6148914690621625735, 12297829383087925879, 18446744073709551615 ],
+          "example_fixed_length_ARRAY_UINT64" : [ 0, 4611686018427387903, 9223372036854775807, 13835058055282163711, 18446744073709551615 ],
+          "example_fixed_length_ARRAY_normalized_UINT64" : [ 0, 4611686018427387903, 9223372036854775807, 13835058055282163711, 18446744073709551615 ],
+          "example_variable_length_ARRAY_FLOAT32" : [ -1.0, -0.3333333334, 0.3333333334, 1.0 ],
+          "example_fixed_length_ARRAY_FLOAT32" : [ -1.0, -0.5, 0.0, 0.5, 1.0 ],
+          "example_variable_length_ARRAY_FLOAT64" : [ -1.0, -0.3333333334, 0.3333333334, 1.0 ],
+          "example_fixed_length_ARRAY_FLOAT64" : [ -1.0, -0.5, 0.0, 0.5, 1.0 ],
+          "example_variable_length_ARRAY_BOOLEAN" : [ true, false, true, false ],
+          "example_fixed_length_ARRAY_BOOLEAN" : [ true, false, true, false, true ],
+          "example_variable_length_ARRAY_STRING" : [ "This", "is", "an", "example" ],
+          "example_fixed_length_ARRAY_STRING" : [ "This", "is", "an", "example", "string" ],
+          "example_variable_length_ARRAY_ENUM" : [ 0, 1, 2, 0 ],
+          "example_fixed_length_ARRAY_ENUM" : [ 0, 1, 2, 0, 1 ]
```

Comment:
Enum values should use the name of the enum value according to the schema: https://github.com/CesiumGS/3d-tiles/blob/main/extensions/3DTILES_metadata/schema/tileset.schema.json#L19

though... does CesiumJS support this detail from the spec? if not we should open an issue.

Reply:
I updated this (and I should have looked it up before creating this example). I was a bit surprised that at no point, any part of the toolchain complained about the wrong type, but maybe that's something that has to be done by a validator.